index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+ <meta charset="utf-8">
+<title>Portfolio</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">   
+<link href="https://fonts.googleapis.com/css?family=Raleway|Ubuntu:400,700&display=swap&subset=latin-ext" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div class="container">
+        <header>
+            <div class="header-nav">
+                <div class="logo"><img src="img/logo.jpg" alt="logo camera"></div>
+                <nav class="header-nav-ul">
+                        <a href="#about-me" class="menu_items">About me</a>
+                        <a href="#skills" class="menu_items">Skills</a>
+                        <a href="#portfolio" class="menu_items">Portfolio</a>
+                        <a href="#contact" class="menu_items">Contact</a>
+                </nav>
+            </div>
+            <div class="header-photo">
+                <h3>Welcome!</h3>
+                <h5>I am a wedding photographer</h5>
+            </div>
+            <div class="header-contact">
+                    <p>I am happy to help you with any problem!</p>
+                    <a href="#" class="header-contact-btn btn-left">Contact me!</a>
+                    <a href="#" class="header-contact-btn btn-right">Check my CV</a>
+            </div>
+        </header>
+        <section id="about-me" class="section-about">
+                <p>About me</p>    
+            <div class="section-about-me">
+                
+                <div class="section-about-me-left">
+                        <img src="img/portretrazem.jpg" alt="portret razem"></div>
```

Comment:
Ładny portret z żoną :D Tylko co to za fotograf robił, jak Wy stoicie po drugiej stronie obiektywu?